config/locales/enumerators/pl.rb

```diff
@@ -0,0 +1,9 @@
+I18n.define_enumerator :pl do
+  def enumerate kind, list, options
+    conjunction = { :all => ' oraz ', :any => ' lub ', :one => ' albo ' }[kind]
+    if 2 >= list.length
+      return list.join conjunction
+    end
+    return list[0...-1].join(', ') + conjunction + list[-1]
+  end
+end
```

Comment:
Can this enumerate method and the english version just delegate to a common enumerate method?  If we end up with 10 languages I wouldn't want 10 copies of this code.  Also, if you can write a spec for the common method, that'd be great.